### PR TITLE
Move MIP-1 Status from Draft to Review

### PR DESCRIPTION
**Pull request by vandan**
Also updated the Motivation section to highlight some additional reasons why using wallet_watchAsset for NFTs would be beneficial to dapp developers.

### CODE REVIEW

File: node_modules/.yarn-integrity [hunk 1]
```diff
@@ -1,10 +0,0 @@
-{
```
Comment (BelfordZ) [inline node_modules/.yarn-integrity:1]: node_modules should be in our .gitignore

Comment (vandan) [reply node_modules/.yarn-integrity:1]: So it should be there, but just .gitignore'd? @shanejonas I thought you were recommending its removal.

Comment (BelfordZ) [reply node_modules/.yarn-integrity:1]: Nothing that you need to do here, just needs to be done as a seperate thing

File: MIPs/mip-1.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ This proposal aims to extend the support of `wallet_watchAsset` in MetaMask to i
...
 
-Furthermore, by expanding `wallet_watchAsset` to support ERC-721 and ERC-1155 tokens, MetaMask will enable users to view and interact with their NFTs directly within the wallet, without relying on third-party indexers or third party APIs. This enhancement empowers users to manage their NFT assets directly, increases the usability of MetaMask for NFT-related activities, and ultimately strengthens user trust in the wallet.
+While MetaMask does offer NFT auto-detection, there limitations in the set of EVM chains that are supported and latency that may be involved. In addition, many users may have NFT auto-detection services disabled due to concerns with potential security and privacy tradeoffs. By expanding `wallet_watchAsset` to support ERC-721 and ERC-1155 tokens, app developers can give their users the ability to view and interact with any of their NFTs directly within MetaMask, without relying on third-party indexers or APIs. This enhancement empowers users to manage their NFT assets directly, increases the usability of MetaMask for NFT-related activities, and ultimately strengthens user trust in the wallet.
```
Comment (adonesky1) [inline MIPs/mip-1.md:18]:
```suggestion
While MetaMask does offer NFT auto-detection currently it is only supported on mainnet. In addition, many users may have NFT auto-detection services disabled due to concerns with potential security and privacy tradeoffs. By expanding `wallet_watchAsset` to support ERC-721 and ERC-1155 tokens, app developers can give their users the ability to view and interact with any of their NFTs directly within MetaMask, without relying on third-party indexers or APIs. This enhancement empowers users to manage their NFT assets directly, increases the usability of MetaMask for NFT-related activities, and ultimately strengthens user trust in the wallet.
```

**Review by adonesky1 — APPROVED** [review]
LGTM

**Pull request merged**